akka-cluster-typed/src/main/scala/akka/cluster/ddata/typed/javadsl/Replicator.scala

```diff
@@ -152,6 +150,11 @@ object Replicator {
    */
   final case class GetFailure[A <: ReplicatedData](key: Key[A]) extends GetResponse[A]
 
+  /**
+   * The [[Get]] request couldn't be performed because the entry has been deleted.
+   */
+  final case class GetDataDeleted[A <: ReplicatedData](key: Key[A]) extends GetResponse[A]
```

Comment:
Wouldn't it be nicer with a common supertype for the non-successful ones so that users doesn't have to cover all of them?

Reply:
Yes

Reply:
Hmm, ok.